modules/tinymce/src/core/main/ts/api/dom/Selection.ts

```diff
@@ -542,11 +544,15 @@ const EditorSelection = (dom: DOMUtils, win: Window, serializer: DomSerializer,
     controlSelection.destroy();
   };
 
+  const expand = (options: { type: 'word' } = { type: 'word' }) =>
+    setRng(RangeUtils(editor.dom).expand(getRng(), options));
```

Comment:
```suggestion
    setRng(RangeUtils(dom).expand(getRng(), options));
```
I missed this in my previous suggestions sorry.